op-node/rollup/derive/batches.go

```diff
@@ -45,36 +63,36 @@ func CheckBatch(cfg *rollup.Config, log log.Logger, l1Blocks []eth.L1BlockRef, l
 	epoch := l1Blocks[0]
 
 	nextTimestamp := l2SafeHead.Time + cfg.BlockTime
-	if batch.Batch.Timestamp > nextTimestamp {
+	if batch.Timestamp > nextTimestamp {
```

Comment:
Why use direct field instead of using `GetTimestamp` wrapper? Applies to other fields of singular batches.